### PR TITLE
[WAF] Rate limits for list operations

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-8065.

### CODE REVIEW

File: content/waf/tools/lists/lists-api/_index.md [hunk 1]
```diff
@@ -18,3 +18,13 @@ To use a list in a rule expression, refer to [Values: Lists](/ruleset-engine/rul
 ## Get started
 
 To get started, review the Lists [JSON object](/waf/tools/lists/lists-api/json-object/) and [Endpoints](/waf/tools/lists/lists-api/endpoints/).
+
+---
+
+## Rate limiting for Lists API requests
+
+Cloudflare may apply rate limiting to Lists API requests creating or updating custom lists and Bulk Redirect Lists.
```
Comment (deadlypants1973) [inline content/waf/tools/lists/lists-api/_index.md]: Should Lists be in backticks? is it like `WebCrypto` API?

File: content/waf/tools/lists/lists-api/_index.md [hunk 2]
```diff
@@ -18,3 +18,13 @@ To use a list in a rule expression, refer to [Values: Lists](/ruleset-engine/rul
...
+
+API requests that create or update items in a list include one or more changes, which are internally queued and then processed. You can have approximately 1,000 pending list changes before your requests are rate limited.
```
Comment (deadlypants1973) [inline content/waf/tools/lists/lists-api/_index.md]: Confused by the first sentence. Does the API request include one or more changes? Or does the list include changes?

Comment (pedrosousa) [reply content/waf/tools/lists/lists-api/_index.md]: Should be `the API request includes one or more changes`. I'll clarify.

File: content/waf/tools/lists/lists-api/_index.md (hunk 1 above)
Comment (pedrosousa) [reply content/waf/tools/lists/lists-api/_index.md]: It's actually the API name, like in "the Web Crypto API".

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**